### PR TITLE
GitHub: Moved github docs into .github folder

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

_N/A_

## Task Checklist

_N/A_

## Steps To Validate Changes

_N/A_

## Known Issues

_N/A_

## PR Checklist

- [ ] **Development**: _N/A_
- [ ] **Approval**: Have discussed with and got approval from the team[^1]
- [x] **Tests**: Tested accessibility on the local end
- [x] **Deployment**: Deployed the app on the local end
- [ ] **Localization**: Modified en-US string resources[^2]
- [ ] **Co-authors**: _N/A_

## Screenshots

_N/A_

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.

### CODE REVIEW

**Comment by 0x5bfa** [comment]
I haven't changed that many lines; just diff have been broken.

**Comment by yaira2** [comment]
> I haven't changed that many lines; just diff have been broken.

Which lines have been changed?

**Comment by 0x5bfa** [comment]
- NOTICE.md
  Nothing. Just moved.
- rich-commands.md
  Added title and description placeholder.

**Review by yaira2 — APPROVED** [review]
LGTM

File: src/Files.App/Files.App.csproj [hunk 1]
```diff
@@ -45,7 +45,7 @@
         <Content Include="7z64.dll">
             <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
         </Content>
-        <Content Include="..\..\NOTICE.md">
+        <Content Include="..\..\.github\NOTICE.md">
```
Comment (yaira2) [inline src/Files.App/Files.App.csproj:48]: Does the about page need to be updated to use this path?

Comment (0x5bfa) [reply src/Files.App/Files.App.csproj:48]: Thank you for having pointed it out. Fixed.

**Review by yaira2 — APPROVED** [review]

**Pull request merged**